code/wdb/datastripe.q

```diff
@@ -83,6 +93,9 @@ upserttopartition:{[dir;tablename;keycol;enumdata;nextp]
         (directory;enumdata);
         {[e] .lg.e[`upserttopartition;"Failed to save table to disk : ",e];'e}
     ];
+
+    /- check if sym file exists in taildir, create sym link to HDB sym file if not
+    if[not `sym in key hsym basedir;createsymlink[.ds.td;.wdb.hdbdir;`sym]];
```

Comment:
this is a little confusing/non-obvious to me - why are we checking for sym in `basedir` but then creating the symlink in `.ds.td`? surely we should be creating the symlink in the same location we checked for it, are `basedir` and `.ds.td` the same? I think we should use the same variable in both places in this line. If they are the same, do we need the local `basedir` variable if this is already stored globally as `.ds.td`?